catalog/spatial/spatial-commands/src/main/java/org/codice/ddf/commands/spatial/gazetteer/GazetteerUpdateCommand.java

```diff
@@ -58,7 +58,8 @@
             + "When the resource is a path to a file, it will be imported locally."
             + "If a path to a file ends in .geo.json, it will processed as a"
             + "geoJSON feature collection and imported as supplementary shape "
-            + "data for geonames entries.",
+            + "data for geonames entries. Additionally, the resource can also be a URL "
+            + "(ex: http://download.geonames.org/export/dump/AI.zip)",
```

Comment:
✏️ Missing a period at the end of this sentence. This description should match the description in the documentation.